src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/ServiceUtil.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.ons.ctp.integration.rhsvc.service.impl;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import java.util.UUID;
+import uk.gov.ons.ctp.common.domain.CaseType;
+import uk.gov.ons.ctp.common.event.EventPublisher;
+import uk.gov.ons.ctp.common.event.EventPublisher.Channel;
+import uk.gov.ons.ctp.common.event.EventPublisher.EventType;
+import uk.gov.ons.ctp.common.event.EventPublisher.Source;
+import uk.gov.ons.ctp.common.event.model.Address;
+import uk.gov.ons.ctp.common.event.model.CollectionCase;
+import uk.gov.ons.ctp.common.event.model.CollectionCaseNewAddress;
+import uk.gov.ons.ctp.common.event.model.NewAddress;
+import uk.gov.ons.ctp.common.time.DateTimeUtil;
+import uk.gov.ons.ctp.integration.rhsvc.representation.CaseRequestDTO;
+
+/**
+ * This class aims to prevent code duplication by holding code which is common to more than one
+ * service.
+ */
+public class ServiceUtil {
```

Comment:
Please make this a final class, with a private constructor (eg using lombok)

Reply:
done